lib/change-log.js

```diff
@@ -28,7 +28,7 @@ export class ChangeLogWindow extends FormApplication {
     let data = await super.getData()
 
     let xhr = new XMLHttpRequest()
-    xhr.open('GET', 'systems/gurps/changelog.md')
+    xhr.open('GET', 'systems/gurps/assets/changelog.md')
```

Comment:
Why this move to assets/?

Reply:
Oh, that's residual from when I previously moved the change log to the src folder. I'll undo it.